lib/src/models/usage.dart

```diff
@@ -41,27 +41,34 @@ class PlanUsageBreakdown {
 
 /// Contains main information about usage.
 class Usage {
+
+  /// End date of a billing cycle.
+  DateTime? billingCycleEnd;
+
+  /// Start date of a billing cycle.
+  DateTime? billingCycleStart;
+
+  /// Indicator if the Explorer has access to a 5 min free call.
+  bool? hasFreeCall;
+
   /// Name of the plan.
   String? planName;
 
-  /// Total available minutes in the plan.
-  int? totalMinutes;
+  /// Indicates if the plan is unlimited.
+  bool? planUnlimited;
 
-  /// Total of used minutes in the plan.
-  int? totalMinutesLeft;
+  /// Indicates if the User is Primary.
+  bool primary;
```

Comment:
One day, we could change this to `isPrimary`. Or maybe replace the `bool` with a `ProfileType`, which already defines `primary` and `secondary`.